### ISSUE TITLE
helium.data.stylesheets is undefined

### ISSUE THREAD

**Issue by welpher** [geuis/helium-css#24]
cause helium.data.stylesheets.sort(); is not a function error

**Comment by ashepherd89** [comment]
Has there been a solution for this problem?  I am having the same problem now.

**Comment by geuis** [comment]
@ashepherd89 Strange, I wasn't able to replicate this before. Would it be possible to see a test case?

**Comment by xeraa** [comment]
Could the problem be that there are no CSS files?

I have a single page site and inline the CSS, so there are no references to CSS files. Getting the same error.

**Comment by geuis** [comment]
@xeraa ahh that might be it. Btw helium doesn't read inline css (since it looks for selectors and style="" on an element isn't something you select for) or style blocks declared on the page.

**Comment by xeraa** [comment]
@geuis yeah — I didn't find a tool that supported inlined CSS and gave up. Plus my HTML+CSS are a little over 17K gzipped, so it's probably not worth the trouble anyway ;)

But nice and simple script in general :)

**Comment by geuis** [comment]
@xeraa When you say "inline", do you mean this:

```
<div style="color: #fff; background: #000"></div>
```

Or this

```
<style>
  div {
    color: #fff;
    background: #000;
  }
</style>
```

**Comment by xeraa** [comment]
The second one; I'm not duplicating the styles, just avoiding a HTTP request (using https://github.com/kaiinui/middleman-inliner).

**Comment by develophelbetica** [comment]
I'm pretty sure I was getting the same error:
"Uncaught ReferenceError: style is not defined"

---> 
I added the following line:
var style = document.createElement('style');
before line 72

```
  var style = document.createElement('style');

  helium.$('body')[0].appendChild(style);
  helium.$('body')[0].appendChild(div);
```

.,. and I'm now getting the URL list box
